astropy/time/formats.py

```diff
@@ -731,17 +731,28 @@ def parse_string(self, timestr, subfmts):
             raise ValueError('Time {0} does not match {1} format'
                              .format(timestr, self.name))
 
+    @staticmethod
+    def _decode_if_needed(val, encoding='ascii'):
+        """Decode ``vall`` if it is a bytes instance."""
+        if type(result) is bytes:
```

Comment:
Err, this should be `val` everywhere not `result`.  This is preventing even import of astropy.